crates/types/src/lib.rs

```diff
@@ -14,6 +14,7 @@ pub mod constants;
 pub mod data;
 pub mod error;
 pub mod event;
+pub mod events;
```

Comment:
It looks like we have `event` and `events`. I think we should either condense these or give them different names

Reply:
Renaming to `hotshot_events` seems reasonable. What do you think?